### ISSUE TITLE
datatables.error = null should respect app.debug & not show exception details to user

### ISSUE THREAD

**Issue by nie7321** [yajra/laravel-datatables#1582]
### Summary of problem or feature request
When the app.debug flag is set to false, SQL errors in the filtering code are causing a popup for the end-user with the exception (in this case, the SQL error).

Per the `config/datatables.php` file, I can control this behaviour pretty easily:

```php
    /**
     * User friendly message to be displayed on user if error occurs.
     * Possible values:
     * null             - The exception message will be used on error response.
     * 'throw'          - Throws a \Yajra\DataTables\Exceptions\Exception. Use your custom error handler if needed.
     * 'custom message' - Any friendly message to be displayed to the user. You can also use translation key.
     */
    'error'          => env('DATATABLES_ERROR', null),
```

I would suggest changing the behaviour if the settings is left `null`: when `app.debug = true`, default to a custom message instead of exposing the exception to an end-user. This would be more consistent with how the rest of the framework behaves.

I can take a stab at a PR if you concur that this is a good idea.

### System details
Laravel 5.5.28, PHP 7.1.8, and Laravel-Datatables 1.0.0

**Comment by yajra** [comment]
Yes, I agree with you. It's actually on my notes but haven't got the chance to implement it. Can you please do submit a PR. TIA ❤️
